VLC 0.8.4 does not work with HAL 0.5 and later, so this patch makes sure that
HAL is not autodetected if it goes over the maximum version allowed.

Index: vlc-0.8.4/configure.ac
===================================================================
--- vlc-0.8.4.orig/configure.ac
+++ vlc-0.8.4/configure.ac
@@ -724,7 +724,7 @@ AC_ARG_ENABLE(hal,
  
 if test "${enable_hal}" != "no" -a "${SYS}" = "linux"
 then
-  PKG_CHECK_MODULES(HAL, hal >= 0.2.97,
+  PKG_CHECK_MODULES(HAL, hal >= 0.2.97 hal < 0.5,
     [AC_DEFINE(HAVE_HAL, [], [Define if you have the HAL library])
      VLC_ADD_PLUGINS([hal])
      VLC_ADD_LDFLAGS([vlc hal],[$HAL_LIBS])
